src/validations/eas-attestation/index.ts

```diff
@@ -0,0 +1,86 @@
+import Validation from '../validation';
+import fetch from 'cross-fetch';
+
+export const author = 'KarmaHQ';
```

Comment:
Hi @andremury PR should be comming from KarmaHQ github fork
```suggestion
export const author = 'andremury';
```

Reply:
Hey, thanks for the feedback. I'm gonna create a PR from there